Foundation/CharacterSet.swift

```diff
@@ -486,7 +486,10 @@ extension CharacterSet : _ObjectTypeBridgeable {
     
     @_semantics("convertToObjectiveC")
     public func _bridgeToObjectiveC() -> NSCharacterSet {
-        return _wrapped
+        switch _wrapped.__wrapped {
```

Comment:
@bubski The new solution sounds reasonable to me. Have you tested locally to confirm the expected behavior?